Fix for Issue #8401

aegisnull opened the pull request:
### Description
This pull request includes the fix for the problem of broken image urls as reported on Issue #8401

### Motivation
This should improve the overall experience of the page by not having broken links on the main content

### Additional details


### Related issues and pull requests
Fixes #8401
https://github.com/mdn/translated-content/issues/8401

<!-- 👷‍♀️ After submitting, go to the "Checks" tab of your PR for the build status -->

Graywolf9 commented:
Hi! @aegisnull thank you so much for your time and contribution. Since the localizated content take the images from en-US content is not necessary to put the images again, it should work only with the changes on the `index.md` file. Could you try it and modify?

aegisnull commented:
@Graywolf9 you have a valid point, as requested I have instead removed the duplicate files I added and replaced the dead links with a link to the images in the original en-US content.

Graywolf9 commented on files/es/web/http/overview/index.md:
```diff
@@ -8,23 +8,24 @@ tags:
 HTTP, de sus siglas en inglés: "Hypertext Transfer Protocol", es el nombre de un protocolo el cual nos permite realizar una petición de datos y recursos, como pueden ser documentos {{glossary("HTML")}}. Es la base de cualquier intercambio de datos en la Web, y un protocolo de estructura cliente-servidor, esto quiere decir que una petición de datos es iniciada por el elemento que recibirá los dato ...
 
-![A Web document is the composition of different resources](https://mdn.mozillademos.org/files/13677/Fetching_a_page.png)
+![A Web document is the composition of different resources](https://developer.mozilla.org/en-US/docs/Web/HTTP/Overview/fetching_a_page.png)
```
The render automatically found the image if it exists in content repo

```suggestion
![A Web document is the composition of different resources](fetching_a_page.png)
```

Graywolf9 commented:
Thank you so much for your attention @aegisnull 

I let you an example, what I mean in my initial comment is that the page automatically take the image from the en-US folder

Graywolf9 commented on files/es/web/http/overview/index.md:
```diff
@@ -8,23 +8,24 @@ tags:
 Clientes y servidores se comunican intercambiando mensajes individuales (en contraposición a las comunicaciones que utilizan flujos continuos de datos). Los mensajes que envía el cliente, normalmente un navegador Web, se llaman _peticiones_, y los mensajes enviados por el servidor se llaman _respuestas_.
 
-![HTTP as an application layer protocol, on top of TCP (transport layer) and IP (network layer) and below the presentation layer.](https://mdn.mozillademos.org/files/13673/HTTP%20&%20layers.png)Diseñado a principios de la década de 1990, {{glossary("HTTP")}} es un protocolo ampliable, que ha ido evolucionando con el tiempo. Es lo que se conoce como un protocolo de la capa de aplicación, y se tran ...
+![HTTP as an application layer protocol, on top of TCP (transport layer) and IP (network layer) and below the presentation layer.](https://developer.mozilla.org/en-US/docs/Web/HTTP/Overview/http-layers.png)Diseñado a principios de la década de 1990, {{glossary("HTTP")}} es un protocolo ampliable, que ha ido evolucionando con el tiempo. Es lo que se conoce como un protocolo de la capa de aplicació ...
```
```suggestion
![HTTP as an application layer protocol, on top of TCP (transport layer) and IP (network layer) and below the presentation layer.](http-layers.png)Diseñado a principios de la década de 1990, {{glossary("HTTP")}} es un protocolo ampliable, que ha ido evolucionando con el tiempo. Es lo que se conoce como un protocolo de la capa de aplicación, y se transmite sobre el protocolo {{glossary("TCP")}}, o el protocolo encriptado {{glossary("TLS")}}, aunque teóricamente podría usarse cualquier otro protocolo fiable. Gracias a que es un protocolo capaz de ampliarse, se usa no solo para transmitir documentos de hipertexto ({{glossary("HTML")}}), si no que además, se usa para transmitir imágenes o vídeos, o enviar datos o contenido a los servidores, como en el caso de los formularios de datos. {{glossary("HTTP")}} puede incluso ser utilizado para transmitir partes de documentos, y actualizar páginas Web en el acto.
```

Graywolf9 commented on files/es/web/http/overview/index.md:
```diff
@@ -8,23 +8,24 @@ tags:
 Cada petición individual se envía a un servidor, el cuál la gestiona y responde. Entre cada _petición_ y _respuesta_, hay varios intermediarios, normalmente denominados {{glossary("Proxy_server", "proxies")}}, los cuales realizan distintas funciones, como: gateways o {{glossary("Cache", "caches")}}.
 
-![Client server chain](https://mdn.mozillademos.org/files/13679/Client-server-chain.png)
+![Client server chain](https://developer.mozilla.org/en-US/docs/Web/HTTP/Overview/client-server-chain.png)
```
```suggestion
![Client server chain](client-server-chain.png)
```

Graywolf9 commented on files/es/web/http/overview/index.md:
```diff
@@ -137,7 +136,7 @@ Existen dos tipos de mensajes HTTP: peticiones y respuestas, cada uno sigue su p
 Un ejemplo de petición HTTP:
 
-![A basic HTTP request](https://mdn.mozillademos.org/files/13687/HTTP_Request.png)
+![A basic HTTP request](https://developer.mozilla.org/en-US/docs/Web/HTTP/Overview/http_request.png)
```
```suggestion
![A basic HTTP request](http_request.png)
```

Graywolf9 commented on files/es/web/http/overview/index.md:
```diff
@@ -151,7 +150,7 @@ Una petición de HTTP, está formado por los siguientes campos:
 Un ejemplo de repuesta:
 
-![](https://mdn.mozillademos.org/files/13691/HTTP_Response.png)
+![](https://developer.mozilla.org/en-US/docs/Web/HTTP/Overview/http_response.png)
```
```suggestion
![](http_response.png)
```

Graywolf9 commented:
Thank you so much for your time and contribution @aegisnull 

I apply the changes to the urls to keep the short way to use, this to improve de load time and follow the guidelines.
